environment.yml

```diff
@@ -0,0 +1,17 @@
+name: occ
+channels:
+  - conda-forge # pythonocc-core
+  - defaults
+dependencies:
+  - python=3.8
+  - anaconda
```

Comment:
what is this for?

Reply:
oh boy, that's probably crap... nice catch...